tests/test_timeout.py

```diff
@@ -0,0 +1,118 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Benchmark for microgrid data."""
```

Comment:
Nitpick: Are these benchmarks, or just timing tests?

Reply:
I'm 99% sure the comment is wrong, and this is just testing that if the server takes too long a timeout error will be raised. I will fix.

Reply:
Fixed.

Reply:
This is likely me. \o/